velox/connectors/hive/storage_adapters/hdfs/CMakeLists.txt

```diff
@@ -19,7 +19,11 @@ add_library(velox_hdfs RegisterHdfsFileSystem.cpp)
 if(VELOX_ENABLE_HDFS)
   target_sources(velox_hdfs PRIVATE HdfsFileSystem.cpp HdfsReadFile.cpp
                                     HdfsWriteFile.cpp)
-  target_link_libraries(velox_hdfs Folly::folly ${LIBHDFS3})
+  target_link_libraries(velox_hdfs Folly::folly ${LIBHDFS3} xsimd)
+
+  if(NOT VELOX_DISABLE_GOOGLETEST)
```

Comment:
this should not be required after rebasing with main.

Reply:
Let me test this by synchronizing the main branch code

Reply:
I have synchronized the latest code, but I still get an error when compiling without this.
```
In file included from /Users/wypb/data/code/apache/temp/velox/velox/connectors/hive/storage_adapters/hdfs/RegisterHdfsFileSystem.cpp:23:
In file included from /Users/wypb/data/code/apache/temp/velox/./velox/dwio/common/DataSink.h:23:
In file included from /Users/wypb/data/code/apache/temp/velox/./velox/dwio/common/DataBuffer.h:22:
In file included from /Users/wypb/data/code/apache/temp/velox/./velox/buffer/Buffer.h:27:
In file included from /Users/wypb/data/code/apache/temp/velox/./velox/common/memory/Memory.h:39:
In file included from /Users/wypb/data/code/apache/temp/velox/./velox/common/memory/Allocation.h:24:
/Users/wypb/data/code/apache/temp/velox/./velox/common/base/GTestMacros.h:24:10: fatal error: 'gtest/gtest_prod.h' file not found
#include <gtest/gtest_prod.h>
         ^~~~~~~~~~~~~~~~~~~~
1 warning and 1 error generated.
```

Reply:
I fixed the previous error. But this is new. This is ugly. I will fix this in a separate PR.

Reply:
Hi @kgpai Thank you for your reply.
I manually set VELOX_DISABLE_GOOGLETEST to ON in hdfs/CMakeLists.txt, then compiled velox_hdfs, and found that the error was the same as above, and the gtest/gtest_prod.h file could not be found

Reply:
@majetideepak Ok, let me synchronize the latest code and compile the project.